home/map.asm

```diff
@@ -1640,6 +1640,8 @@ ReturnToMapWithSpeechTextbox::
 	ld [wSpriteUpdatesEnabled], a
 	call ClearBGPalettes
 	call ClearSprites
+	farcall ClearSavedObjPals
```

Comment:
After FIQ and ariscop were discussing `farcall`s, I wonder if this adds any perceptible delay to the overworld textboxes.

Reply:
Perhaps the discussion in discord has overstated the problem; they're on the order of 10x the overhead of a plain call/ret, which isn't that bad in practice

ReloadTilesetAndPalettes already does a lot of farcalls, a few more is unlikely to be noticeable